codemod/base.py

```diff
@@ -25,6 +25,8 @@
 import sys
 import textwrap
 from math import ceil
+if os.uname()[0] == 'Darwin':
+    from spotlight_util import *
```

Comment:
No wildcard imports please. Import what you need, or just `import spotlight_util`.